src/utils/soroban/getConfig.ts

```diff
@@ -1,22 +1,34 @@
-import { Contract, SorobanRpc } from 'stellar-sdk';
+import { Contract, Keypair, SorobanRpc } from 'stellar-sdk';
 
-import getAdmin from './getAdmin';
+import { Network } from '../../types/networkType';
 
-const getConfig = async (network: string) => {
-  let contract = new Contract(String(process.env.MAINNET_CONTRACT_ID));
-  let server = new SorobanRpc.Server(String(process.env.MAINNET_FUTURENET_RPC_URL));
-  let admin = await server.getAccount(getAdmin().publicKey());
+const getConfig = async (network: Network) => {
+  const adminSecretKey = Keypair.fromSecret(String(process.env.ADMIN_SECRET_KEY));
```

Comment:
This is not admin's secret key. This is admin keypair. I suggest renaming it to 'adminKeypair'